how-we-work/tracking-time.md

```diff
@@ -1,20 +1,21 @@
 # Tracking time
 
 A paramount aspect of our work is our ability to make good use of the time we spend at the office. 
-To spend time efficiently, measuring it how we spend it is essential.
+Measuring is essential in order to spend it efficiently.
 
 There are two different measures of how we spend our time:
 
 * hours of _presence_ in the office;
-* hours of _work_ on client/company projects.
+* hours of _active work_ on projects.
```

Comment:
Can we change active work with activity? I'd like just use one word to make it as simple as possible and avoid any confusion.

Reply:
I'm still not fully convinced on either of those, "activity" is broad, but we can go with it… if nothing else, it has the good quality of conciseness 👍